pkg/cgroups/manager/manager_test.go

```diff
@@ -178,6 +179,8 @@ func TestGetPodMetadataOnPodAdd(t *testing.T) {
 }
 
 func TestGetPodMetadataOnPodUpdate(t *testing.T) {
+	setup(t)
```

Comment:
Thanks a lot.  

The previous test (e.g. TestGetPodMetadataOnPodAdd) performs required setup when this entire file is run in CI.

Reply:
I think it's best to avoid any side-effect on setup step, so if you need to push one more time, can you add the below patch as  well

```diff
diff --git a/pkg/cgroups/manager/manager_test.go b/pkg/cgroups/manager/manager_test.go
index f19e2346ef..826bed6862 100644
--- a/pkg/cgroups/manager/manager_test.go
+++ b/pkg/cgroups/manager/manager_test.go
@@ -128,6 +128,11 @@ func newCgroupManagerTest(pMock providerMock, cg cgroup) *CgroupManager {
 func setup(tb testing.TB) {
 	option.Config.EnableSocketLBTracing = true
 	nodetypes.SetName("n1")
+
+	tb.Cleanup(func() {
+		option.Config.EnableSocketLBTracing = false
+		nodetypes.SetName("localhost")
+	})
 }
 
 func getFullPath(path string) string {
```

Reply:
The common patter I've seen is 

```
func TestFoo(t *Testing) {
    setup()
    defer cleanup()
     ...
}
```
Anyway it's not required here. All tests need the setup to be called, except one that explicitly disables the config.